### DISCUSSION TITLE
Plot tables from arrays or pandas dataframes

### DISCUSSION THREAD

**Discussion by AlexStox** [highfestiva/finplot#111] [General]
Hi Jonas, I've read in the Finplot doc that it can also plot not only time-series. Can it plot a table? For instance from a data frame? I saw some solution in matplot but first would like to hear from you whether it's possible in Finplot.

**Comment by highfestiva** [comment]
Hi Aladdin! No sorry, that is not possible out of the box as finplot is built for backtesting. finplot has PyQt under the hood, so it's probably not super-hard to generate some type of grid wherein you fill in columns, index and cells, and so forth. On the other hand you can just print the data frame...
